modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/portlet/action/ExportTranslationMVCResourceCommand.java

```diff
@@ -124,7 +125,8 @@ public boolean serveResource(
 					_getZipFileName(
 						translationRequestHelper.getModelClassName(),
 						translationRequestHelper.getModelClassPK(),
-						sourceLanguageId, _portal.getLocale(resourceRequest)),
+						SetUtil.fromArray(classPKs), sourceLanguageId,
```

Comment:
It seems you're passing SetUtil.fromArray(classPKs) just to get the number of elements. Why not pass the size of the array directly? Alternatively, you could compute the default name outside _getZipFileName and pass it directly to the method.

Reply:
The side of the array can be different than the number of classPKs
If we select the languages French and Chinese for different models you will have` number of languages x number of models`, thats why I put it on a Set to remove the duplicates

Reply:
Sorry, when I said "... pass the size of the array directly" I meant the size of the set.

Reply:
ah, ok, that was only because I was trying to avoid chaining :)

Reply:
> Alternatively, you could compute the default name outside _getZipFileName and pass it directly to the method.

I do not see any advantage of extracting the default name previously of the method,  because we have to recalculate the almost same logic outside and inside the method.


From my point of view, the only logic thing is to extract classNameTitle that can be used on two places

Reply:
But could we at least pass the number of elements instead of the set? I'm fine with the current use of the names.